./run ide watch does not work on recent develop

Today I faced that `./run ide watch` stopped working on my machine.

```
TypeError: process.on is not a function
 INFO ide_ci::program::command: npm⚠️     at file:///Users/ilyabogdanov/projects/enso/app/ide-desktop/lib/client/watch.ts:79:9
 INFO ide_ci::program::command: npm⚠️ npm ERR! Lifecycle script `watch` failed with error:
 INFO ide_ci::program::command: npm⚠️ npm ERR! Error: command failed
 INFO ide_ci::program::command: npm⚠️ npm ERR!   in workspace: enso@0.0.0-dev
 INFO ide_ci::program::command: npm⚠️ npm ERR!   at location: /Users/ilyabogdanov/projects/enso/app/ide-desktop/lib/client
TRACE ide_ci::program::command: npmℹ️ <ENDUT>
```

Command I'm running: `./run ide watch --backend-source=release --backend-release="2023.1.1-nightly.2023.3.4" --cargo-watch-option -w --cargo-watch-option ../rebuild_me --wasm-size-limit=0 --skip-wasm-opt`

Last part of the log: https://gist.github.com/vitvakatu/252b3d230221a0bedefe248fc8c047c6

Version: `a6487e51055d88de53ba4aedac6416c85f155d6e`

--- a/app/ide-desktop/eslint.config.js
+++ b/app/ide-desktop/eslint.config.js
@@ -21,9 +21,10 @@ const NAME = 'enso'
  * and conversely type errors may not mean they don't support ESM -
  * but we add those to the whitelist anyway otherwise we get type errors.
  * In particular, `string-length` supports ESM but its type definitions don't.
- * `yargs` and `react-hot-toast` are modules we explicitly want the default imports of. */
+ * `yargs` and `react-hot-toast` are modules we explicitly want the default imports of.
+ * `node:process` is here because `process.on` does not exist on the namespace import. */
 const DEFAULT_IMPORT_ONLY_MODULES =
-    'chalk|string-length|yargs|yargs\\u002Fyargs|sharp|to-ico|connect|morgan|serve-static|create-servers|electron-is-dev|fast-glob|esbuild-plugin-alias|esbuild-plugin-time|esbuild-plugin-yaml|opener'
+    'node:process|chalk|string-length|yargs|yargs\\u002Fyargs|sharp|to-ico|connect|morgan|serve-static|create-servers|electron-is-dev|fast-glob|esbuild-plugin-alias|esbuild-plugin-time|esbuild-plugin-yaml|opener'
 const ALLOWED_DEFAULT_IMPORT_MODULES = `${DEFAULT_IMPORT_ONLY_MODULES}|react-hot-toast`
 const OUR_MODULES = 'enso-content-config|enso-common'
 const RELATIVE_MODULES =

--- a/app/ide-desktop/lib/client/src/index.ts
+++ b/app/ide-desktop/lib/client/src/index.ts
@@ -8,7 +8,7 @@
 import * as fs from 'node:fs/promises'
 import * as fsSync from 'node:fs'
 import * as pathModule from 'node:path'
-import * as process from 'node:process'
+import process from 'node:process'
 
 import * as electron from 'electron'
 

--- a/app/ide-desktop/lib/client/watch.ts
+++ b/app/ide-desktop/lib/client/watch.ts
@@ -12,7 +12,7 @@
 import * as childProcess from 'node:child_process'
 import * as fs from 'node:fs/promises'
 import * as path from 'node:path'
-import * as process from 'node:process'
+import process from 'node:process'
 
 import * as esbuild from 'esbuild'
 

--- a/app/ide-desktop/utils.ts
+++ b/app/ide-desktop/utils.ts
@@ -1,7 +1,7 @@
 /** @file Shared utility functions. */
 import * as fs from 'node:fs'
 import * as path from 'node:path'
-import * as process from 'node:process'
+import process from 'node:process'
 
 /**
  * Get the environment variable value.

